### DISCUSSION TITLE
About changing the design of the underline that appears under text

### DISCUSSION THREAD

**Discussion by initOki** [zed-industries/zed#18532] [Feature Requests]
Can I change the design of the underline that appears under problematic text while writing code? It's currently coming out as a wave, but I'd like to change it to a straight line.
![Screenshot 2024-09-30 at 7 51 05 PM](https://github.com/user-attachments/assets/9f969b7a-3e33-48de-bf54-d0b62b4f9fe4)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
